buildSrc/src/main/groovy/nva.identity.service.jacoco-merge.gradle

```diff
@@ -1,66 +0,0 @@
-//For generating one unified coverage report
```

Comment:
File not needed anymore?

Reply:
Gradle 8 seems to handle jacocomerge differently.